apps/challenges/views.py

```diff
@@ -67,6 +67,8 @@
     is_user_in_allowed_email_domains,
     is_user_in_blocked_email_domains,
     parse_submission_meta_attributes,
+    add_domain,
+    add_tags,
```

Comment:
Rename it `add_domain_to_challenge`, `add_tags_to_challenge`, I will merge it after this.

Reply:
Done